public/app/features/dashboard/components/DashExportModal/DashboardExporter.ts

```diff
@@ -177,7 +177,8 @@ export class DashboardExporter {
           model = libPanel.model;
         }
 
-        const { gridPos, id, ...rest } = model;
+        // eslint-ignore
+        const { gridPos, id, ...rest } = model as any;
```

Comment:
Not sure how this works since we're explicitly omitting these from the model? Also type assertion isn't necessary if there's an ignore rule. 
```suggestion
        const { gridPos, id, ...rest } = model;
```

Reply:
the "model" here is not actually constrained by the typescript -- since it gets set directly from a full PanelModel.

These lines remove the values we do not want to save on export.  Removing the "as any" makes this fail: 
<img width="770" alt="image" src="https://user-images.githubusercontent.com/705951/214879846-f9b90d29-583d-421f-a1f1-e39888d53daa.png">

Reply:
I see, in that case the `// eslint-ignore` should not be necessary?

Reply:
I removed it -- i hoped it would avoid the betterer warning.